### PR TITLE
Document the localization class and method removal breaking change

### PR DESCRIPTION
**Pull request by scottaddie**
Document the breaking change described at https://github.com/aspnet/Announcements/issues/407.

### CODE REVIEW

File: includes/core-changes/aspnetcore/5.0/localization-members-removed.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+#### Reason for change
+
+The [ResourceManagerWithCultureStringLocalizer](/dotnet/api/microsoft.extensions.localization.resourcemanagerwithculturestringlocalizer?view=dotnet-plat-ext-3.1) class and [ResourceManagerStringLocalizer.WithCulture](/dotnet/api/microsoft.extensions.localization.resourcemanagerstringlocalizer.withculture?view=dotnet-plat-ext-3.1) method were often sources of confusion for users of localization. The confusion was especially high when creating a custom <xref:Microsoft.Extensions.Localization.IStringLocalizer> implementation. This class and method give consumers the impression that an `IStringLocalizer` instance is expected to be "per-language, per-resource". In reality, the instance should only be "per-resource", with the language searched for determined by the <xref:System.Globalization.CultureInfo.CurrentUICulture%2A?displayProperty=nameWithType> property at execution time.
```
Comment (gewarren) [inline includes/core-changes/aspnetcore/5.0/localization-members-removed.md:23]:
```suggestion
The [ResourceManagerWithCultureStringLocalizer](/dotnet/api/microsoft.extensions.localization.resourcemanagerwithculturestringlocalizer?view=dotnet-plat-ext-3.1) class and [ResourceManagerStringLocalizer.WithCulture](/dotnet/api/microsoft.extensions.localization.resourcemanagerstringlocalizer.withculture?view=dotnet-plat-ext-3.1) method were often sources of confusion for users of localization. The confusion was especially high when creating a custom <xref:Microsoft.Extensions.Localization.IStringLocalizer> implementation. This class and method give consumers the impression that an `IStringLocalizer` instance is expected to be "per-language, per-resource". In reality, the instance should only be "per-resource". At execution time, the <xref:System.Globalization.CultureInfo.CurrentUICulture%2A?displayProperty=nameWithType> property determines the language that's searched for.
```

I suggested a rewrite of the last sentence. Also, should "execution time" be "run time"?

File: includes/core-changes/aspnetcore/5.0/localization-members-removed.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+For context, see [aspnet/Announcements#346](https://github.com/aspnet/Announcements/issues/346) and [dotnet/aspnetcore#3324](https://github.com/dotnet/aspnetcore/issues/3324). For discussion on this change, see [dotnet/aspnetcore#7756](https://github.com/dotnet/aspnetcore/issues/7756).
+
+#### Version introduced
+
+5.0 Preview 1
+
+#### Old behavior
+
+The `ResourceManagerWithCultureStringLocalizer` class and the `ResourceManagerStringLocalizer.WithCulture` method are obsolete in .NET Core 3.0 Preview 3 and later.
```
Comment (gewarren) [inline includes/core-changes/aspnetcore/5.0/localization-members-removed.md:15]: Link to the 3.0 breaking change?

**Review by gewarren — APPROVED** [review]

**Comment by scottaddie** [comment]
@gewarren @tdykstra @BillWagner Are we comfortable with merging this PR? The build passes, but there are content validation issues that are out of our control.

**Comment by gewarren** [comment]
Yes, I'm comfortable merging it.

**Pull request merged**